### ISSUE TITLE
As a user, I want to have translation on belarusian and english languages

### ISSUE THREAD

**Issue by rizomaa** [diglabby/mapa#5]

**Comment by fr0zen** [comment]
#1

**Comment by fr0zen** [comment]
- [ ] Create UI to switch between locales.
- [ ] Create addition fields in DB, that contains localized titles of initiatives.
- [ ] Change form of initiative creation to be able to enter localized version of title.

**Comment by rizomaa** [comment]
Need axplanation:
1. "Create UI to switch between locales."

Has this been already done?

2. " Change form of initiative creation to be able to enter localized version of title."

Need more explanation.

3. "Think how titles will be shown on initiative's marks on map."

Need more explanation. Please show

**Comment by fr0zen** [comment]
1. I mean some button or checkbox, that will be a trigger to switch between languages.
2. We need Belarussian and english version of titles for every organisation.

**Comment by wellemut** [comment]
The interface is in English already!

Interface-Translation is easy:
Just use this file: https://github.com/flosse/kartevonmorgen/blob/master/src/locales/translation-de.json


1. Copy it to an text editor
2. Translate allways the second part (after the : ) Never change the first word bevor : , Never forgett the " ..." around the word and the comma at the end of each line 
"**Keyword**":"**This text you can translater into your language**",

Thank you very much!
You finde all language files in this folder, where we will upload your translation. https://github.com/flosse/kartevonmorgen/tree/master/src/locales

**Comment by rizomaa** [comment]
@wellemut yes, we know that english's been already done. we mostly focus here on priorities we expect.

**Comment by wellemut** [comment]
I redefined this issue as bilingual content, because the interface-multilingauality is done allready and just needs belarussian translation.

**Comment by fr0zen** [comment]
- [ ] Get actual be translation by [link](https://github.com/diglabby/mapa/files/3388744/BYloc.be.zip)
- [ ] Merge actual file https://github.com/diglabby/mapa/blob/develop/src/locales/translation-by.json with be translation file provided by previous link.
- [ ] Set belarussian translation by default in https://github.com/diglabby/mapa/blob/develop/src/i18n.js (Now it's set according to browser locale)
- [ ] Syncronize https://github.com/diglabby/mapa/blob/develop/src/locales/translation-en.json array with https://github.com/diglabby/mapa/blob/develop/src/locales/translation-by.json 
- [ ] Add link to switch to english locale in landing interface.

**Comment by rizomaa** [comment]
Here is root issue for testing the feature
https://github.com/diglabby/mapa/issues/303
